### ISSUE TITLE
Make http_client timeout a configurable parameter

### ISSUE THREAD

**Issue by predictiple** [Velocidex/velociraptor#201]
The current 10 second timeout is too restrictive for things like client upgrades, resulting in:

```
vql: scope.go:317: http_client: Reading error net/http: request canceled (Client.Timeout exceeded while reading body)
```

**Comment by scudette** [comment]
Hi, this should have been addressed in the 0.3.7 release. Is it still an
issue?

The timeout now is controlled by the query timeout

**Comment by predictiple** [comment]
Thanks Mike. I can confirm that once I upgrade the client to 0.3.7 it's no longer an issue.

I was using 0.3.7 server-side and attempting to upgrade a 0.3.6 client. This was just for testing purposes so it's not a real problem for me. Glad it will work in future.
